packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -0,0 +1,214 @@
+import type { SendFeedbackParams } from '@sentry/core';
+import { captureFeedback, getCurrentScope, lastEventId, logger } from '@sentry/core';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import {
+  Alert,
+  Image,
+  Keyboard,
+  KeyboardAvoidingView,
+  SafeAreaView,
+  ScrollView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  TouchableWithoutFeedback,
+  View
+} from 'react-native';
+
+import { sentryLogo } from './branding';
+import { defaultConfiguration } from './defaults';
+import defaultStyles from './FeedbackForm.styles';
+import type { FeedbackFormProps, FeedbackFormState, FeedbackFormStyles,FeedbackGeneralConfiguration, FeedbackTextConfiguration } from './FeedbackForm.types';
+
+let feedbackFormHandler: (() => void) | null = null;
+
+const setFeedbackFormHandler = (handler: () => void): void => {
+  feedbackFormHandler = handler;
+};
+
+const clearFeedbackFormHandler = (): void => {
+  feedbackFormHandler = null;
+};
+
+type Navigation = {
+  navigate: (screen: string, params?: Record<string, unknown>) => void;
+};
+
+export const showFeedbackForm = (navigation: Navigation): void => {
+  setFeedbackFormHandler(() => {
+    navigation?.navigate?.('FeedbackForm');
+  });
+  if (feedbackFormHandler) {
+    feedbackFormHandler();
+  } else {
+    logger.error('FeedbackForm handler is not set. Please ensure it is initialized.');
+  }
+};
```

Comment:
This was merged from https://github.com/getsentry/sentry-react-native/pull/4370

---

I would keep the auto inject out of the this PR.

I think we should not export this helper, because to use it user still have to create a screen which will present the `FeedbackForm` component. Only then the helper shows the form.

---

The auto inject should work without manually specifying the FeedbackForm component and should be independent of any nav library. (We can create helper for specific bav libs later...)

I imagine when I call `showFeedbackForm()` the form will just show, no need to setup anything besides `Sentry.init`/`Sentry.wrap`(since we can't inject to dom as on web).

Reply:
Makes sense @krystofwoldrich 👍 
I reverted the change with e7a94844feea6b43a2d7c942c1d8bb5329099f76 and will iterate on the auto inject on a separate PR.